CHANGELOG.md

```diff
@@ -7,6 +7,184 @@ Versions are `MAJOR.PATCH`.
+- Properly discover block devices path in virt.running (#60296)
+- Avoid exceptions when handling some exception cases. (#60330)
+- Fixed faulty error message in npm.installed state. (#60339)
+- Port option reinstated for Junos Proxy (accidentally removed) (#60340)
+- Now hosts.rm_host can remove entries from /etc/hosts when this file have inline comments. (#60351)
+- Fixes issue where the full same name is not used when making rights assignments with group policy (#60357)
+- Fixed zabbix_host.present to not overwrite inventory_mode to "manual" everytime inventory is updated. (#60382)
+- Allowed zabbix_host.present to do partial updates of inventory, also don't erase everything if inventory is missing in state definition. (#60389)
+- Fixing the mysql_cache module to handle binary inserting binary data into the database. Initially adding tests. (#60398)
+- Fixed host_inventory_get to not throw an exception if host does not exist (#60418)
+- Check for /dev/kvm to detect KVM hypervisor. (#60419)
+- Fixing file.accumulated handling of dependencies when the state_id is used instead of {function: state_id} format. (#60426)
+- Adding the ability for yumpkg.remove to handle package names with widdcards. (#60461)
+- Pass emulator path to get guest capabilities from libvirt (#60491)
+- virt.get_disks: properly report qemu-img errors (#60512)
+- Make all platforms have psutils. This prevents a minion from starting if an instance is all ready running. (#60523)
+- Ignore configuration for 'enable_fqdns_grains' for AIX, Solaris and Juniper, assume False (#60529)
+- Remove check for TIAMAT_BUILD enforcing USE_STATIC_REQUIREMENTS, this is now controled by Tiamat v7.10.1 and above (#60559)
+- Have the beacon call run through a try...except, catching any errors, logging and firing an event that includes the error.
+  Fixing the swapusage beacon to ensure value is a string before we attempt to filter out the %. (#60585)
+- Refactor loader into logical sub-modules (#60594)
+- Clean up references to ZMQDefaultLoop (#60617)
+- change dep warn from Silicon to Phosphorus for the cmd,show,system_info and add_config functions in the nxos module. (#60669)
+- Fix bug 60602 where the hetzner cloud provider isn't recognized correctly (#60675)
+- Fix the ``pwd.getpwnam`` caching issue on macOS user module (#60676)
+- Fixing beacons that can include a value in their configuration that may or may not included a percentage.  We want to handle the situation where the percentage sign is not included and the value is not handled as a string. (#60684)
+- Fix RuntimeError in process manager (#60749)
+- Ensure all data that is being passed along to LDAP is in an OrderedSet and contains bytes. (#60760)
+- Update the AWS API version so VMs spun up by salt-cloud where the VPC has it enabled to assign ipv6 addresses by default, actually get ipv6 addresses assigned by default. (#60804)
+- Remove un-needed singletons from tranports (#60851)
+- Deprecate `salt.payload.Serial` (#60953)
```

Comment:
This one should actually be in the deprecation section. I missed the proper file extensions while reviewing the PR